src/cpp/dbnode.cpp

```diff
@@ -30,29 +30,27 @@
 
 using namespace SmartRedis;
 
+// DBNode constructor
 DBNode::DBNode()
+    : ip(""), port(-1), name(""), lower_hash_slot(-1), upper_hash_slot(-1)
 {
-   this->name = "";
-   this->ip = "";
-   this->port = -1;
-   this->lower_hash_slot = -1;
-   this->upper_hash_slot = -1;
+   // NOP
 }
 
-DBNode::DBNode(std::string ip, std::string name,
-               uint64_t port, uint64_t l_slot,
-               uint64_t u_slot, std::string prefix)
+// DBNode constructor with connection and hash slot information.
+DBNode::DBNode(std::string _ip, std::string _name, uint64_t _port,
+               uint64_t l_slot, uint64_t u_slot, std::string _prefix)
+    : ip(_ip), port(_port), name(_name), lower_hash_slot(l_slot),
```

Comment:
I don't think we should start passing in variables with ``_`` prefix because we have removed ``this``.  I think that's bad example to set for future developers.  Can we think of better solution?

Reply:
I'm not sure where the confusion is -- this is standard practice for constructors to differentiate between the variable supplied to the constructor and the member variable. C++ scoping rules say that locally scoped variables have first priority for a name. Second priority (which is the case here) is the parameters to a function. In the case of initializers, there can't be any locally scoped variables, so these are the top priority. Let's discuss offline.